src/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+// eslint-disable-next-line
+import ReactDOM from "react-dom/client";
+import './index.css';
+import { Provider } from 'react-redux';
+import App from './App';
+// eslint-disable-next-line
+import store from "./redux/configureStore";
+
```

Comment:
- [ ] The way you are importing the `store` object is incorrect. The correct path should be:

`"./redux/ConfigureStore"`. In your path for the import statement, you are using lowercase _**c**_ instead of uppercase _**C**_ for the `configureStore` word. This causes the error I attached in the screenshot below


![image](https://user-images.githubusercontent.com/77106962/217838618-fc955ffa-6c5b-492b-90ef-75b641438ef4.png)

#### Kindly fix this error 👍 😎